src/app/containers/Metadata/index.test.jsx

```diff
@@ -10,6 +12,15 @@ import { RequestContextProvider } from '../../contexts/RequestContext';
 import frontPageData from '../../../../data/igbo/frontpage/index.json';
 import liveRadioPageData from '../../../../data/korean/bbc_korean_radio/liveradio.json';
 
+const MockHelmet = ({ children }) => <>{children}</>;
```

Comment:
Not a review just a question, what is a react helmet?

Reply:
@HarveyPeachey React Helmet is this package - https://github.com/nfl/react-helmet. It enables us to pass React elements from a component to be rendered in the head of the document (in the `<head></head>` tag) rather than in the body. All of the `<meta>` elements within a `<Helmet>` get pulled into the head tag within the Document component - see here: https://github.com/bbc/simorgh/blob/latest/src/server/Document/component.jsx#L20